src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,31 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    enum Colors {
```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s4.8.1-enum-classes
fix naming

Reply:
move enum to a separate file